documentation/index.rst

```diff
@@ -180,6 +180,7 @@ The platform operator of FlexMeasures can be an Aggregator.
     :maxdepth: 1
     
     tut/toy-example-from-scratch
```

Comment:
Maybe we could reorder this to show the `tut/installation` tutorial first?

Reply:
@Nischay-Pro made the same suggestion in a current PR of his.